### ISSUE TITLE
Link to this repo from official docs

### ISSUE THREAD

**Issue by ash211** [apache-spark-on-k8s/spark#34]
https://spark.apache.org/docs/latest/ -> Deploying -> Kubernetes

Experimental
Unsupported by Apache project
Community-maintained

**Comment by ash211** [comment]
Blocked on https://github.com/apache-spark-on-k8s/spark/issues/35

**Comment by foxish** [comment]
Do we want to move our changes to the master branch in this repo before linking from the docs?

**Comment by lins05** [comment]
SGTM. BTW I suggest rebasing our main branch with the master branch regularly (e.g. once a week) to avoid possible big surprise in the end.

**Comment by ash211** [comment]
I'm not sure if master branch is where we want to be for wider usage vs off a release branch.  An alternate approach from ~weekly master branch merges could be to create a `branch-2.1-kubernetes` based off Apache's `branch-2.1` and do our work on that.  Then when 2.2 is released we forward port the diff to `branch-2.2` and continue on from there.

This is all assuming we don't go the plugin approach though, which is seeming more likely.

**Comment by lins05** [comment]
>  An alternate approach from ~weekly master branch merges could be to create a branch-2.1-kubernetes based off Apache's branch-2.1 and do our work on that. Then when 2.2 is released we forward port the diff to branch-2.2 and continue on from there.

+1. It makes sense to keep up with the current release branch instead of the master branch.

**Comment by erikerlandson** [comment]
I'm in favor of `branch-2.1-kubernetes` as well.   We don't want confusion with `master`.   We can continue rebasing, which should remain pretty clean.

**Comment by foxish** [comment]
I was apprehensive about having multiple release branches, because we may end up in a situation where we're supporting integrations and patches in multiple branches. Given that the changes mostly don't affect the core, that may be okay. We're close to finishing up phase 1, what do you think of creating an alpha release and then opening the floodgates (linking from official docs).

**Comment by ash211** [comment]
I think we're really close -- I'll start a new ticket with a burndown list to first broad announce

**Comment by ash211** [comment]
https://github.com/apache-spark-on-k8s/spark/issues/61

**Comment by foxish** [comment]
This is done in https://github.com/apache/spark/pull/17522, closing.
